liquibase.yaml

```diff
@@ -0,0 +1,183 @@
+package:
+  name: liquibase
+  version: "4.31.1"
+  epoch: 0
+  description: "Liquibase is a database schema change management solution that enables you to revise and release database changes faster and safer from development to production."
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    runtime:
+      - bash
+      - liquibase-package-manager
+      - openjdk-${{vars.java-version}}-default-jvm
+
+vars:
+  java-version: '17'
+
+environment:
+  contents:
+    packages:
+      - bash
+      - build-base
+      - busybox
+      - maven
+      - openjdk-${{vars.java-version}}-default-jdk
+      - wolfi-base
+  environment:
+    LANG: en_US.UTF-8
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/liquibase/liquibase
+      tag: v${{package.version}}
+      expected-commit: 12ab20b176e5659622c240b4ba4f056ccf17c27d
+
+  - name: Setup
+    runs: |
+      # Reversion liquibase.build.properties
+      # REF: https://github.com/liquibase/liquibase/blob/master/.github/util/re-version.sh#L87
+      sed -i -e "s/build.version=.*/build.version=${{package.version}}/" liquibase-standard/src/main/resources/liquibase.build.properties
+
+  - name: Build
+    runs: |
+      # Build liquibase
+      ./mvnw -B clean package -DskipTests=true -P !liquibase-commercial
+
+  - uses: strip
+
+subpackages:
+  - name: "${{package.name}}-compat"
+    description: "Compat package for Liquibase Symlinks"
+    pipeline:
+      - runs: |
+          # Setup destination dir
+          mkdir -p ${{targets.destdir}}/usr/share/java/liquibase
+          tar -zxvf liquibase-dist/target/${{package.name}}-0-SNAPSHOT.tar.gz -C ${{targets.destdir}}/usr/share/java/liquibase
+          cp docker-entrypoint.sh ${{targets.destdir}}/usr/share/java/liquibase
+          chmod +x ${{targets.destdir}}/usr/share/java/liquibase/docker-entrypoint.sh
+          cp liquibase.docker.properties ${{targets.destdir}}/usr/share/java/liquibase
+
+          # Symlinks for liquibase
+          mkdir -p ${{targets.contextdir}}/usr/local/bin
+          ln -sf /usr/share/java/liquibase ${{targets.contextdir}}/liquibase
+          ln -sf /usr/share/java/liquibase/liquibase ${{targets.contextdir}}/usr/local/bin/liquibase
+          ln -sf /usr/share/java/liquibase/docker-entrypoint.sh ${{targets.contextdir}}/usr/local/bin/docker-entrypoint.sh
+
+          # Symlinks for java
+          mkdir -p ${{targets.contextdir}}/opt/java/openjdk
+          for item in bin conf legal lib release; do
+            ln -sf /usr/lib/jvm/java-17-openjdk/$item ${{targets.contextdir}}/opt/java/openjdk
+          done
```

Comment:
Remove this, and in the main package just do `ln -s /usr/lib/jvm/default-jvm ${{targets.destdir}}/opt/java/openjdk`